### ISSUE TITLE
[Page] Events, when no upcoming events

### ISSUE THREAD

**Issue by megamos** [Kodkollektivet/kodkollektivet.se#39]
Currently there is no text/info at all when there is no upcoming event.
Lets add something. Like info about things that occur regularly, such as codhubs.

Or maybe a whole new section on the events page about our CodeHubs?
(Everybody doesn't use facebook.)

**Comment by jherrlin** [comment]
Maybe we should try to get these events from FB, would that be possible?

**Comment by ritzelpaket** [comment]
Yes. I'll write a script for the job. Should be included in Django as a schedules job.

How must the data be prepared? JSON? How will the data be stored in Django?

**Comment by jhe10709** [comment]
I think we could use http://docs.celeryproject.org/en/latest/django/first-steps-with-django.html for this. We have a model for events here https://github.com/Kodkollektivet/kodkollektivet.se/blob/master/kodkollektivet/kodkollektivet/models.py#L38 , guess it would make sense to use that. Or what do you think?

**Comment by jhe10709** [comment]
@ritzelpaket is the script in Python?

**Comment by ritzelpaket** [comment]
The script will be written in Python. I will write the script and fetch some data. Then I will have a look if it fits the model. Maybe we need to add some fields to the model, if it is not too much work.

**Comment by jhe10709** [comment]
Modifying the model is no problem and easy to do! I think it can be really easy to save it to the DB if you have a json object with key names that representing the model.

**Comment by jhe10709** [comment]
@ritzelpaket hows it going with this? should we make some effort together?

**Comment by ritzelpaket** [comment]
Yes! Did you come forward with your guile implementation? Or should we finally write it in python? I will try to fix the fb dev account this week.

**Comment by jhe10709** [comment]
At the moment I am working on a Guile implementation but its not done, wanna help out on that or should we go for a Python implementation? I got some help from my colleagues with Facebooks Graph API.

**Comment by ritzelpaket** [comment]
The python script to fetch caldav events is working now. We can successfully build event objects in JSON now. How and were should we store them then?

**Comment by ritzelpaket** [comment]
I really like this calendar library: https://fullcalendar.io/. Could be used to display events on the desktop version. I think mobile should be a different library, because I think it is not responsive. Or maybe the list view for mobile.

**Comment by AhmadVakil** [comment]
@ritzelpaket The JSON export will be finished soon and then we can save them into any database. We can get it done soon.
